experiments/conductor/cmd/runner/mock_commands.go

```diff
@@ -436,3 +438,41 @@ func addProtoToMakfile(opts *RunnerOptions, branch Branch) {
 	gitCommit(workDir, &out, fmt.Sprintf("Adding proto to Makefile for %s", branch.Name))
 
 }
+
+func runMockgcpTests(opts *RunnerOptions, branch Branch) {
+	close := setLoggingWriter(opts, branch)
+	defer close()
+	workDir := filepath.Join(opts.branchRepoDir, "mockgcp")
+
+	var out strings.Builder
+	checkoutBranch(branch, workDir, &out)
+
+	// Check to see if the http log file already exists
+	logFile := fmt.Sprintf("mock%s/testdata/%s/crud/_http.log", branch.Group, branch.Resource)
+	logFullPath := filepath.Join(workDir, logFile)
+
+	// Run the test against the generated mocks to determine quality
+	start := time.Now()
+	log.Printf("COMMAND: go test ./mockgcptests -v -run TestScripts/mock%s/testdata/%s/crud", branch.Group, branch.Resource)
```

Comment:
Would be more useful to comment this way
WRITE_GOLDEN_OUTPUT=1 E2E_GCP_TARGET=mock go test .....

Reply:
I'm note sure, there are two parts to this one, the command log and the actual command.

For the log I'm not sure if its a good idea to represent the logged command differently than the execute command.

For the actual command I think probably not. 
"A=B command" is a shell trick and I'm not sure that shell trick will work here.
Eg. "egrep foo *.txt" does not work in exec because the * expansion is done bu the shell not the command. 
Also I think environment variables should be passed in through the environment API.
I think it would be better to change mockgcptests to take anything non-sensitive as a command line flag.